libs/librtlnumber/src/rtl_int.cpp

```diff
@@ -44,6 +46,9 @@ static compare_bit eval_op(VNumber& a_in, VNumber& b_in) {
     assert_Werr(b_in.size(),
                 "empty 2nd bit string");
 
+    if (a_in.is_dont_care_string() || b_in.is_dont_care_string())
```

Comment:
@jeanlego @aarongraham9 I would be interested in hearing your guys opinion here. Technically the spec say that this is correct. However, this has a performance impact since now we have to loop through the string an extra time.
My questions are:
1) Does the performance matter here, since we are usually only going to be up to 32 bits. Though I imagine this code might be quite hot
2) Do we care about exact conformance to the spec
3) What do you think about having a has_unknown bool in VNumber so we do not have to loop.

(Apologies if this is a duplicate, I lost my first comment on my screen)

Reply:
Yeah I think this is reasonable. Go for it! :)